piper.in

```diff
@@ -7,6 +7,10 @@ import sys
 gi.require_version('Gio', '2.0')
 gi.require_version('Gtk', '3.0')
 from gi.repository import Gio, Gtk
+gtk_version = Gtk.get_major_version(), Gtk.get_minor_version()
+if gtk_version < (3, 22):
```

Comment:
Should we maybe move the detection into meson.build and replace this with `if gtk_version < @gtk_version@:` so we have only meson.build in which we track dependencies? Not sure if this is at all possible, just thinking out loud.